app/script/media/MediaStreamHandler.js

```diff
@@ -445,13 +437,26 @@ z.media.MediaStreamHandler = class MediaStreamHandler {
    * Initial request for local MediaStream was successful.
    *
    * @private
+   * @param {string} conversationId - ID of conversation to initiate MediaStream for
    * @param {z.media.MediaStreamInfo} mediaStreamInfo - Type of requested MediaStream
    * @returns {undefined} No return value
    */
-  _initiateMediaStreamSuccess(mediaStreamInfo) {
+  _initiateMediaStreamSuccess(conversationId, mediaStreamInfo) {
     if (mediaStreamInfo) {
+      const conversationNeedsMedia = this.currentCalls.get(conversationId);
```

Comment:
Do we have the guarantee that the `conversationId` is the same as the `callId`?
Cause here, we compare call ids with conversation ids.

Reply:
Yes